datafusion/sqllogictest/test_files/timestamps.slt

```diff
@@ -2910,6 +2910,299 @@ query error DataFusion error: Arrow error: Cast error: Cannot cast string '' to
 select make_date(2024, 1, '');
 
 
+##########
+## make_time tests
+##########
+
+query D
+select make_time(14, 30, 45);
+----
+14:30:45
+
+query D
+select make_time(0, 0, 0);
+----
+00:00:00
+
+query D
+select make_time(23, 59, 59);
+----
+23:59:59
+
+query D
+select make_time('14', '30', '45');
+----
+14:30:45
+
+query D
+select make_time(12 + 2, '30', '45');
+----
+14:30:45
+
+query D
+select make_time(14::bigint, 30::bigint, 45::bigint);
+----
+14:30:45
+
+query D
+select make_time(arrow_cast(14, 'Int64'), arrow_cast(30, 'Int64'), arrow_cast(45, 'Int64'));
+----
+14:30:45
+
+query D
+select make_time(arrow_cast(14, 'Int32'), arrow_cast(30, 'Int32'), arrow_cast(45, 'Int32'));
+----
+14:30:45
+
+statement ok
+create table table_times (hour int, minute int, second int) as values
+    (14, 30, 45),
+    (9, 15, 30);
+
+query D
+select make_time(t.hour, t.minute, t.second) from table_times t;
+----
+14:30:45
+09:15:30
+
+query D
+select make_time(12, t.minute, t.second) from table_times t;
+----
+12:30:45
+12:15:30
+
+query D
+select make_time(t.hour, 0, t.second) from table_times t;
+----
+14:00:45
+09:00:30
+
+query D
+select make_time(t.hour, t.minute, 0) from table_times t;
+----
+14:30:00
+09:15:00
+
+query D
+select make_time('12', t.minute, t.second) from table_times t;
+----
+12:30:45
+12:15:30
+
+query D
+select make_time(t.hour, '0', t.second) from table_times t;
+----
+14:00:45
+09:00:30
+
+query D
+select make_time(t.hour, t.minute, '0') from table_times t;
+----
+14:30:00
+09:15:00
+
+statement ok
+insert into table_times values (14, null, 45);
+
+query error DataFusion error: Execution error: Unable to parse time from 14, 0, 45
```

Comment:
Should this fail ?
Apache Spark return NULL if any of the arguments is NULL - https://github.com/apache/spark/blob/7251e95e22cc9afd39bcae3ad0ef56b7843ac0fb/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/timeExpressions.scala#L538-L539